telegram/ext/filters.py

```diff
@@ -171,6 +173,33 @@ def filter(self, message):
     command = _Command()
     """:obj:`Filter`: Messages starting with ``/``."""
 
+    class regex(BaseFilter):
```

Comment:
Code for regex filter is good (regardless of some nitpicking).
However, it has nothing to do with the deep linking PR.
Please open a separate PR for it.

Reply:
See comment